src/coreclr/vm/frames.h

```diff
@@ -1964,12 +1964,12 @@ typedef DPTR(class ComMethodFrame) PTR_ComMethodFrame;
 // This represents a generic call from CLR to COM
 //------------------------------------------------------------------------
 
-class ComPlusMethodFrame : public FramedMethodFrame
+class CRLToCOMMethodFrame : public FramedMethodFrame
```

Comment:
`CRLToCOM` -> `CLRToCOM`

Reply:
You might want to slow down a bit on this one. This isn't a simple Find/Replace exercise. Reading the code around the changes will help to confirm if there are other places to update.

Reply:
Okay, I'll take a closer look. Do you have any examples or tips? I'm new to runtime

Reply:
It seems that the current version is not building neither on pipeline nor locally since I pulled the last main commit from yesterday. I've thoses errors:

C:\Dev\runtime2\src\coreclr\System.Private.CoreLib\src\System\RuntimeType.CoreCLR.cs(3970,17): error CS0305: Using the generic type 'GenericCache<TKey, TValue>' requires 2 type argume 
nts [C:\Dev\runtime2\src\coreclr\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Dev\runtime2\src\coreclr\System.Private.CoreLib\src\System\RuntimeType.CoreCLR.cs(3972,29): error CS0122: 'RuntimeType.ActivatorCache.ActivatorCache(RuntimeType)' is inaccessible d 
ue to its protection level [C:\Dev\runtime2\src\coreclr\System.Private.CoreLib\System.Private.CoreLib.csproj]
C:\Dev\runtime2\src\coreclr\System.Private.CoreLib\src\System\RuntimeType.CoreCLR.cs(3973,17): error CS0305: Using the generic type 'GenericCache<TKey, TValue>' requires 2 type argume 
nts [C:\Dev\runtime2\src\coreclr\System.Private.CoreLib\System.Private.CoreLib.csproj]

It seems to be related to the function `internal void CallDefaultStructConstructor(ref byte data)` inside `RuntimeType.CoreCLR.cs`

Reply:
Honestly touching the native part isn't that handy for new contributors. It requires understanding into the codebase, and also some knowledge about interop.

For this case, these types are the types for interop, but their names are inappropriate. The new proposed name `ClrToComCall` provides a clear indication about relation with interop and the direction.

Reply:
It's clearer. I think that for the first classes, this refactor will do the trick. For future renames, I'll have to look a little harder at how it works.